README.md

```diff
@@ -47,3 +39,43 @@ The options include:
 
 Regardless of the workflow chosen, it is important that we strive to keep the git history clean.
 This means never force pushing on main, and usually having one commit per new post.
+
+## Install
```

Comment:
Would it make sense to change the title to something like: "Testing the generated content" or something like that?